conf/application.conf

```diff
@@ -3,3 +3,7 @@ db.default.driver="org.postgresql.Driver"
 db.default.url="jdbc:postgresql://localhost:5500/bakerydb"
 db.default.username="postgres"
 db.default.password="pass"
+# Test config
+db.test.url="jdbc:postgresql://localhost:5550/bakerydb_test"
```

Comment:
if you are setting up a different database configuration you are asking your users to start the database container some other way, right? Does this mean your docker command needs updating?